ucca/layer1.py

```diff
@@ -433,12 +433,12 @@ def next_id(self):
             except KeyError:
                 return id_str
 
-    def add_fnode(self, parent, edge_tag, *, implicit=False):
+    def add_fnode(self, parent, edge_categories, *, implicit=False):
         """Adds a new :class:`FNode` whose parent and Edge tag are given.
 
         :param parent: the FNode which will be the parent of the new FNode.
                 If the parent is None, adds under the layer head FNode.
-        :param edge_tag: the tag on the Edge between the parent and the new FNode.
+        :param edge_categories: list of categories on the Edge between the parent and the new FNode.
```

Comment:
Again, document the exact expected format of the parameter (tuples or Category objects)